tests/gem/test_packets.py

```diff
@@ -13,7 +13,7 @@
     serial_number=123456,
     seconds=0,
     pulse_counts=[0] * packets.GEMPacketFormat.NUM_PULSE_COUNTERS,
-    temperatures=[0] * packets.GEMPacketFormat.NUM_TEMPERATURE_SENSORS,
+    temperatures=list([0.0] * packets.GEMPacketFormat.NUM_TEMPERATURE_SENSORS),
```

Comment:
I can't really explain why wrapping this in `list` helps, but it does make pyright happy.

Reply:
It was both, actually.  `0.0` was needed because this is a `float`, not `int` `List`.  `list` was needed because pyright didn't realize it was a `List` (but rather `list` with a lowercase `l`).